### PR TITLE
Doc-173-Update-IAM-Policy-Examples

### PR DESCRIPTION
**Pull request by Siboney1**
Minimum static and dynamic permissions update

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/775#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by resmaa** [comment]
@Siboney1 @truegilb 
these policies need to be kept on hold as per the inputs from @sadysnaat (Deepak Sharma)

File: content/docs/04-clusters/1-new-clusters/01-aws.md
Comment (truegilb) [inline content/docs/04-clusters/1-new-clusters/01-aws.md:626]: Can we check with engineering to update a date - 2012 is 10 years ago

File: content/docs/04-clusters/1-new-clusters/01-aws.md
Comment (truegilb) [inline content/docs/04-clusters/1-new-clusters/01-aws.md:762]: Can we check with engineering to update a date - 2012 is 10 years ago

**Review by truegilb — CHANGES_REQUESTED** [review]
Please refer to inline comments.

File: content/docs/04-clusters/1-new-clusters/01-aws.md
Comment (sadysnaat) [reply content/docs/04-clusters/1-new-clusters/01-aws.md:762]: This is fine. I think the schema version for AWS json/iam roles

**Review by sadysnaat — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1114702381" title="Approved">sadysnaat</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@Siboney1: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | b7a833d9e4dde49c77df3d8f48a638acd73d62a2 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/775/librarium_build/1575618465673252864) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/1-new-clusters/01-aws.md
Comment (truegilb) [reply content/docs/04-clusters/1-new-clusters/01-aws.md:762]: Thanks @sadysnaat for the confirmation.

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1114702381" title="Approved">sadysnaat</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1126115639" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sadysnaat — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1126153380" title="Approved">sadysnaat</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1126115639" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1126153380" title="Approved">sadysnaat</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/775#pullrequestreview-1126115639" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **zulfilee**
You can assign the PR to them by writing `/assign @zulfilee` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**
- **[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/master/prow/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Siboney1** [comment]
Closing PR and opening a new PR with a clean slate as this PR is behind several commits behind the head. There is too many to reconcile.

**Pull request closed without merging**
